cms/djangoapps/contentstore/rest_api/v2/views/tests/test_downstreams.py

```diff
@@ -433,6 +433,45 @@ def test_204_no_upstream(self, mock_sever):
         assert response.status_code == 204
         assert mock_sever.call_count == 1
 
+    def test_unlink_parent_should_update_children(self):
+        """
+        If we unlink a parent block, do all children get unlinked as well?
+        """
```

Comment:
I think the name and the docstring should reflect that you are testing that the children have the updated top-level parent:

```suggestion
    def test_unlink_parent_should_update_children_top_level_parent(self):
        """
        If we unlink a parent block, do all children get the new top-level parent?
        """
```